docs/API/Elements.md

```diff
@@ -5,6 +5,24 @@ title: Elements
 
 The Elements API is a powerful abstraction that allows users to manipulate any property on any element through a handful of simple methods. This is important because these few methods allow for easy networking or persistent state. 
 
+When it comes to Javascript Element objects each element has a few public properties. These can be used to reference the object.
+
+```js
+var a = this.id;   // Read-only. Gets the element's ID
```

Comment:
I'd add additional spacing between each piece of code to keep everything consistent on the page stylistically (made a similar comment down below)